### ISSUE TITLE
Sonarr : Profile ID (language) deprecated ?

### ISSUE THREAD

**Issue by Pherscho** [vche/pycliarr#33]
I have updated to Sonarr 4.0 and when I try to add a series I get an error.  In the profile id that is now showing "Deprecated" it just to show "English".

```
`Select the item to add (1-100):1
[1]: Any (['SDTV', 'WEB 480p', 'DVD', 'HDTV-720p', 'HDTV-1080p', 'WEB 720p', 'Bluray-720p', 'WEB 1080p', 'Bluray-1080p'])
[2]: SD (['SDTV', 'WEB 480p', 'DVD'])
[3]: HD-720p (['HDTV-720p', 'WEB 720p', 'Bluray-720p'])
[4]: HD-1080p (['HDTV-1080p', 'WEB 1080p', 'Bluray-1080p'])
[5]: Ultra-HD (['HDTV-2160p', 'WEB 2160p', 'Bluray-2160p'])
[6]: HD - 720p/1080p (['HDTV-720p', 'HDTV-1080p', 'WEB 720p', 'Bluray-720p', 'WEB 1080p', 'Bluray-1080p'])
Profile id to use (1-6):6
[1]: Deprecated
Profile id to use (1-1):1
1 // 1 // 0
API error: Error from server http://localhost:8989/api/v3/series, status: 400, msg: ('{\n'
 '  "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",\n'
 '  "title": "One or more validation errors occurred.",\n'
 '  "status": 400,\n'
 '  "traceId": "00-7c2c53d2f607225ddeb6c191a02bf74b-c332097bf37b3613-00",\n'
 '  "errors": {\n'
 '    "": [\n'
 '      "The supplied value is invalid."\n'
 '    ]\n'
 '  }\n'
 '}')`
```

**Comment by vche** [comment]
Thanks for raising this, i will fix this in the upcoming version implementing the custom defaults.

**Comment by vche** [comment]
Fixed in release 1.0.23
